pkgs/development/node-packages/default.nix

```diff
@@ -340,6 +340,19 @@ let
       '';
     };
 
+    reveal-md = super.reveal-md.override (
+      lib.optionalAttrs (!stdenv.isDarwin) {
```

Comment:
What happens if I try to build the package on Darwin?

Reply:
It builds successfully for me. I found this pattern for other node apps based on puppeteer in default.nix

I don't know why it doesn't work on linux, but on darwin we can't use the chromium package.

Reply:
See [mermaid-cli](https://github.com/NixOS/nixpkgs/blob/5775aa907655ce9ee2a2e6470a9b3a124dcc3351/pkgs/development/node-packages/default.nix#L257)